README.md

```diff
@@ -34,14 +34,24 @@ Edit this document to include your answers after each question. Make sure to lea
 
 1. If you were to describe semantic HTML to the next cohort of students, what would you say?
 
+Semantic HTML is a tyoe of coding that is very useful because it not only gives hints as to what each line is reffering to but it also makes it easier for the browser and text-readers to find and explain the content of the code on the page.
```

Comment:
great definition! while humans can distinguish things base on visual formatting, seach engines/browsers/screen readers all have to rely on markup (in this case semantic html) to see the meaning of the content